django/db/migrations/state.py

```diff
@@ -323,6 +323,16 @@ def rename_field(self, app_label, model_name, old_name, new_name):
                         for from_field_name in from_fields
                     ]
                 )
+            # Fix field names (e.g. for CompositePrimaryKey) to refer to the
+            # new field.
+            if field_names := getattr(field, "field_names", None):
```

Comment:
I guess we'll want to eventually have a `composite` flag on `CompositeField` (e.g. like we have with `many_to_many` and friends) but that's the best we can do right now.